lib/physiqual/exporters/exporter.rb

```diff
@@ -1,17 +1,21 @@
 module Physiqual
   module Exporters
     class Exporter
-      def export_data(user, last_measurement_time, from, to)
-        services = create_services(user.physiqual_tokens, last_measurement_time)
-        # TODO: Should the set of imputers used be configurable too?
+      def export_data(user, first_measurement, number_of_days)
+        services = create_services(user.physiqual_tokens)
         data_aggregator = DataAggregator.new(services, Physiqual.imputers)
 
+        from = first_measurement - Physiqual.hours_before_first_measurement.hours
+        to   = first_measurement + (number_of_days - 1).days +
+               ((Physiqual.measurements_per_day - 1) * Physiqual.interval).hours
+
         activities = data_aggregator.activities(from, to)
         heart_rate = data_aggregator.heart_rate(from, to)
         steps = data_aggregator.steps(from, to)
         calories = data_aggregator.calories(from, to)
 
         result = {}
+        # TODO: Waarom index je dit op de dates steps en itereer je niet gewoon over alle dates?
```

Comment:
yeah dit was een beetje de quick fix. We hebben hier die lijst met dates niet toch? dan moeten we ze opnieuw berekenen.